src/Command/Ide/IdeListMineCommand.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Acquia\Cli\Command\Ide;
+
+use AcquiaCloudApi\Endpoints\Applications;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Helper\TableSeparator;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class IdeListMineCommand.
+ */
+class IdeListMineCommand extends IdeCommandBase {
+
+  protected static $defaultName = 'ide:list-mine';
+
+  /**
+   * {inheritdoc}.
+   */
+  protected function configure() {
+    $this->setDescription('List available Cloud IDEs belonging to you');
+    $this->acceptApplicationUuid();
+  }
+
+  /**
+   * @param \Symfony\Component\Console\Input\InputInterface $input
+   * @param \Symfony\Component\Console\Output\OutputInterface $output
+   *
+   * @return int 0 if everything went fine, or an exit code
+   * @throws \Exception
+   */
+  protected function execute(InputInterface $input, OutputInterface $output) {
+    $acquia_cloud_client = $this->cloudApiClientService->getClient();
+    $account_ides = $acquia_cloud_client->request('get', '/account/ides');
```

Comment:
Opened an upstream issue to add this endpoint: https://github.com/typhonius/acquia-php-sdk-v2/issues/208

Not a blocker for this PR, but feel free to weigh in there.